tools/PrebidValidator/Dr.Prebid/Settings/SettingsTableViewController.m

```diff
@@ -173,6 +173,69 @@ -(void)tableView:(UITableView *)tableView willDisplayCell:(UITableViewCell *)cel
         [cell setLayoutMargins:UIEdgeInsetsZero];
     }
     
+    if(indexPath.section == 0 && indexPath.row == 0 && [cell isKindOfClass:[SegmentCell class]]){
```

Comment:
Why moving these from didPressNext to here?

Reply:
Becuase the scroll of page prevents the 1st cell values to be persisted. So basically the values present in screen only is stored & not all the scrolled up values